javaslang-benchmark/src/test/java/javaslang/collection/IteratorBenchmark.java

```diff
@@ -40,33 +39,31 @@ public static void main(String... args) {
         Integer[] ELEMENTS;
 
         /* Only use these for non-mutating operations */
```

Comment:
but if it's mutable, can't we use a simple `ArrayList` in the background, instead of a linked list?